orchagent/portsorch.cpp

```diff
@@ -1432,6 +1433,7 @@ bool PortsOrch::addPort(const set<int> &lane_set, uint32_t speed, int an, string
     }
 
     m_portListLaneMap[lane_set] = port_id;
+    m_portCount++;
```

Comment:
>m_portCount [](start = 4, length = 11)

It is assigned in `PortsOrch::PortsOrch()` and unchanged later. Do you intend to change the semantics?